Lib/json/tool.py

```diff
@@ -55,7 +55,7 @@ def main():
             infile = open(options.infile, encoding='utf-8')
         try:
             if options.json_lines:
-                objs = (json.loads(line) for line in infile)
+                objs = tuple(json.loads(line) for line in infile)
```

Comment:
This has significant memory implications for large `.jsonl` files.  It might be better to instead move the `finally` clause either to the outer `try` or just after the `with outfile` block, but that does have implications for the `infile == outfile` case.

Reply:
Hmm, yeah, moving it gives a blank file when `infile == outfile`...